test/utils/env.go

```diff
@@ -8,13 +8,15 @@ const (
 	platformChannel = "PLATFORM_CHANNEL"
 	serviceChannel  = "SERVICE_CHANNEL"
 	localSnap       = "LOCAL_SNAP"
+	fullConfigTest  = "FULL_CONFIG_TEST"
 )
 
 var (
 	// global defaults
 	PlatformChannel = "latest/edge"
 	ServiceChannel  = "latest/edge"
 	LocalSnap       = ""
+	FullConfigTest  = "off"
```

Comment:
Please use boolean true/false. The env var would still be string "true"/"false" and coverted to boolean in `init()`.

Reply:
After removal of the extra start/restart calls in the last PR, how long does it take now to run all tests locally?

Reply:
device-mqtt test : decrease from (testing...) to 346s
all tests: (testing...)